sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AnalysisWarningsSensor.java

```diff
@@ -41,31 +42,29 @@
 import java.util.regex.Pattern;
 import java.util.stream.Stream;
 
-public class AnalysisWarningsSensor implements Sensor {
+@Phase(name = Phase.Name.POST)
+public final class AnalysisWarningsSensor implements ProjectSensor {
```

Comment:
Similar with [ZeroCoverageSensor](https://github.com/SonarSource/sonarcloud-core/blob/825c070ec138dfb95be2104d386b910d5db0b78e/sonar-scanner-engine/src/main/java/org/sonar/scanner/source/ZeroCoverageSensor.java#L21)

Reply:
Maybe add a comment here, like the `UnitTestResultsImportSensor` has. To make sure the future us know what the intention was.